db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek_informed_consent.rb

```diff
@@ -23,7 +23,7 @@
         </li>
         <li>Alle onderzoeksgegevens worden naar strikte ethische richtlijnen en met grote voorzichtigheid behandeld. Al mijn antwoorden op de vragen worden anoniem opgeslagen en zijn niet tot mijn persoon te herleiden. De onderzoekers gebruiken cookies bij de online vragenlijst om te zien hoeveel mensen op de website van de vragenlijst komen en hoe zij de vragenlijst gebruiken. Deze cookies bevatten geen persoonsgegevens, en volgen me niet naar andere websites.
         </li>
-        <li>Het staat mij vrij om naam en e-mailadres op te geven aan het einde van het onderzoek, die zijn nodig om aanspraak te maken op een bol.com cadeaubon ter waarde van €10,-. Deze persoonsgegevens worden apart van de onderzoeksdata opgeslagen en deze twee vormen van data zijn niet aan elkaar te koppelen. Het is mij duidelijk dat het opgeven van mijn persoonsgegevens geheel vrijwillig is: ik mag er ook voor kiezen om deze gegevens niet in te vullen. In dat geval kan ik geen beloning ontvangen, maar wel meedoen aan het onderzoek. 
+        <li>Het staat mij vrij om mijn e-mailadres op te geven aan het einde van het onderzoek, indien ik hiervoor kies wordt ook mijn IP-adres opgeslagen. Dit is nodig om eenmalig aanspraak te kunnen maken op een bol.com cadeaubon ter waarde van €10,-. Deze persoonsgegevens worden apart van de onderzoeksdata opgeslagen en deze twee vormen van data zijn niet aan elkaar te koppelen. Het is mij duidelijk dat het opgeven van mijn e-mailadres geheel vrijwillig is: ik mag er ook voor kiezen om dit niet in te vullen. In dat geval kan ik geen beloning ontvangen, maar wel meedoen aan het onderzoek.
```

Comment:
dit klopt niet, die data zijn wel te koppelen. Wat beter is:

`deze twee vormen van data zijn voor de onderzoekers niet te koppelen`.

Daarnaast vraag ik me nog een beetje af waarom we nou perse die IP adressen willen opslaan.

Reply:
Die reden waarom we het willen opslaan moet in ieder geval duidelijk in het plan staan, mbt avg eisen.

Reply:
Mee eens.

Reply:
Ik heb aanpassingen voorgesteld in de google docs en overlegd met mandy.

Reply:
Fixed.